.github/workflows/ci.yml

```diff
@@ -546,6 +547,31 @@ jobs:
           secret_vault_secretId: ${{ secrets.VAULT_SECRET_ID }}
           secret_vault_address: ${{ secrets.VAULT_ADDR }}
           secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
+  
+  openapi-lint:
+    name: C8 REST OpenAPI linting
+    timeout-minutes: 10
```

Comment:
You could consider lowering this to reflect more accurately how much time is reasonably needed (few minutes I guess).